vllm_ascend/patch/worker/patch_0_8_4/patch_metrics.py

```diff
@@ -15,19 +15,13 @@
 # limitations under the License.
 #
 
-from typing import Callable, Optional, Union
+from typing import Optional, Union
 
 import torch
 import torch_npu
 from vllm.spec_decode.metrics import (AsyncMetricsCollector,
                                       SpecDecodeWorkerMetrics)
 
-Timer = Callable[[], float]
```

Comment:
I found the Timer here is useless @shen-shanshan please double check it. Thanks.